### DISCUSSION TITLE
Changing the timer input method

### DISCUSSION THREAD

**Discussion by aryamm20** [SelfControlApp/selfcontrol#766] [Ideas]
Hi,
This is an extremely minor QoL improvement request but could the slider for the timer please be change so that it either has tickers (and therefore it is easier to sift through the different minutes), or  has a manual duration input option so that one can quickly input the length of their block duration? It obviously works perfectly at the moment, it's just a bit finicky.

Thank you!

**Comment by moverton** [comment]
I'd consider this a significant QoL issue. Please consider changing the input method for setting the duration. Using a range slider to set an exact number is extraordinarily difficult. All that's needed is a simple time input -- no adjusting of rotating dials, no manipulation of clock hands, nothing extra that requires a high-precision effort. A user should be able to do it entirely from a keyboard.

**Comment by dovima77** [comment]
Is there a way to to set the time to always on with out the need to set a time limit?

**Reply by slambert** [reply]
No, that's outside the scope of the application. But [read the FAQ](https://github.com/SelfControlApp/selfcontrol/wiki/FAQ) and you may find another solution to the problem.
